### ISSUE TITLE
Deprecated dependencies in @mapbox/node-pre-gyp

### ISSUE THREAD

**Issue by Johnwjl** [mapbox/node-pre-gyp#881]
- Operating System (or Browser): macOS
- Node Version: 20.17.0
- Link to reproduction: https://stackblitz.com/edit/nuxt-starter-m7qsfv?file=README.md 

### Description

I noticed that `@mapbox/node-pre-gyp@1.0.11` introduces deprecated dependencies:

- `npmlog@5.0.1`
   - `are-we-there-yet@2.0.0`
   - `gauge@3.0.2`
- `rimraf@3.0.2`
   - `glob @7.2.3`

Could you please consider updating `npmlog` `rimraf` and addressing these deprecated dependencies?

### Steps to reproduce

1. Install a project that depends on `@mapbox/node-pre-gyp`
2. Run `pnpm install`
3. Observe deprecation warnings about these packages.

Dependency chain:
```sh
nuxt 3.13.1
└─┬ nitropack 2.9.7
  └─┬ @vercel/nft 0.26.5
    └─┬ @mapbox/node-pre-gyp 1.0.11
      └─┬ npmlog 5.0.1
        └── are-we-there-yet 2.0.0
        └── gauge 3.0.2
      └── rimraf 3.0.2
         └── glob 7.2.3
```

**Comment by heath-freenome** [comment]
These deprecated dependencies are triggering Snyk warnings in our enterprise. Given that `rimraf` is an easy upgrade, can't you at least make that change? Not sure how hard `npmlog` upgrade is

**Comment by Tofandel** [comment]
They seem to have addressed them already but haven't published a release yet

**Comment by leeobrum** [comment]
Do we have any solution?

**Comment by leeobrum** [comment]
can you help us @lukekarrys?

**Comment by minenwerfer** [comment]
I hope this issue gets solved, it's spamming the console with warning messages when my package is installed.

**Comment by cclauss** [comment]
What is the status of this?

**Comment by HummingMind** [comment]
+1
Would also like to see this get fixed.

Thank you! 🍻

**Comment by benmccann** [comment]
I believe this should be fixed in 2.0.0-rc.0. `npmlog` was replaced with `consola`. `rimraf` is no longer a direct dependency, but unfortunately is still pulled in by `minizlib`. There are a couple PRs out to remove it such as https://github.com/isaacs/minizlib/pull/30. But I think it's a newer version so it should at least fix the deprecation warnings

**Comment by HummingMind** [comment]
Looks like the deprication warnings are gone with 2.0.0-rc.0.

Thank you!
